satellite/orders/service.go

```diff
@@ -707,6 +707,72 @@ func (service *Service) CreatePutRepairOrderLimits(ctx context.Context, bucketID
 	return limits, piecePrivateKey, nil
 }
 
+// CreateGracefulExitPutOrderLimit creates an order limit for graceful exit put transfers.
+func (service *Service) CreateGracefulExitPutOrderLimit(ctx context.Context, bucketID []byte, nodeID storj.NodeID, pieceNum int32, rootPieceID storj.PieceID, shareSize int32) (limit *pb.AddressedOrderLimit, _ storj.PiecePrivateKey, err error) {
+	defer mon.Task()(&ctx)(&err)
+
+	orderExpiration := time.Now().Add(service.orderExpiration)
```

Comment:
should this be time.Now().UTC() or is it okay to use time.Now()?

Reply:
I prefer UTC. Updated